src/keyboard/Key.tsx

```diff
@@ -9,6 +10,28 @@ interface KeyProps {
   onClick?: () => void;
 }
 
+interface BehaviorShortName {
+  short?: string | null;
+}
+
+const shortNames: Record<string, BehaviorShortName> = BehaviorShortNames;
+
+const shortenHeader = (header: string | undefined) => {
+  if(typeof header === "undefined"){
+    return "";
+  }
+  const maxHeaderLength = 9;
```

Comment:
I personally don't love constants hiding in functions, Can we pull this out so it's easier to find, and maybe capitalize, e.g. `MAX_HEADER_LENGTH` ?